Dockerfile

```diff
@@ -63,15 +63,16 @@ RUN GROUP_NAME=$(getent group $USER_GID | awk -F':' '{print $1}') \
 
 ### RUBY
 
-# Install Ruby 2.6.6, update RubyGems, and install Bundler
+# Install Ruby 2.7, update RubyGems, and install Bundler
 ENV BUNDLE_SILENCE_ROOT_WARNING=1
 RUN apt-add-repository ppa:brightbox/ruby-ng \
   && apt-get update \
-  && apt-get install -y ruby2.6 ruby2.6-dev \
+  && apt-get install -y ruby2.7 ruby2.7-dev \
   && gem update --system 3.2.20 \
+  && gem uninstall bundler --version 2.1.4 \
```

Comment:
Is this still needed now that we `bundle exec`?

Reply:
Probably not 👍